localstack/services/ec2/provider.py

```diff
@@ -316,6 +324,59 @@ def describe_transit_gateways(
             transit_gateway["Options"].update(tgw.options)
         return result
 
+    @handler("CreateLaunchTemplate", expand=False)
+    def create_launch_template(
+        self,
+        context: RequestContext,
+        request: CreateLaunchTemplateRequest,
+    ) -> CreateLaunchTemplateResult:
+
+        # parameter validation
+        if not request["LaunchTemplateData"]:
+            raise MissingParameterError(parameter="LaunchTemplateData")
+
+        name = request["LaunchTemplateName"]
+        if len(name) < 3 or len(name) > 128 or not re.fullmatch(r"[a-zA-Z0-9.\-_()/]*", name):
+            raise InvalidLaunchTemplateNameError()
+
+        return call_moto(context)
+
+    @handler("ModifyLaunchTemplate", expand=False)
+    def modify_launch_template(
+        self,
+        context: RequestContext,
+        request: ModifyLaunchTemplateRequest,
+    ) -> ModifyLaunchTemplateResult:
+
+        backend = ec2_backends[context.account_id][context.region]
+        template_id = (
+            request["LaunchTemplateId"]
+            or backend.launch_template_name_to_ids[request["LaunchTemplateName"]]
+        )
+        template: MotoLaunchTemplate = backend.launch_templates[template_id]
+
+        # check if defaultVersion exists
+        if request["DefaultVersion"]:
+            try:
+                template.versions[int(request["DefaultVersion"]) - 1]
+            except IndexError:
+                raise CommonServiceException(
```

Comment:
nit: Could also be defined as a separate exception subclass.

Reply:
+1